openmc/mgxs/library.py

```diff
@@ -54,9 +54,10 @@ class Library(object):
         If true, computes cross sections for each nuclide in each domain
     mgxs_types : Iterable of str
         The types of cross sections in the library (e.g., ['total', 'scatter'])
-    domain_type : {'material', 'cell', 'distribcell', 'universe'}
+    domain_type : {'material', 'cell', 'distribcell', 'universe', 'mesh'}
         Domain type for spatial homogenization
-    domains : Iterable of openmc.Material, openmc.Cell or openmc.Universe
+    domains : Iterable of openmc.Material, openmc.Cell, openmc.Universe or
```

Comment:
Where would these lines show up in the sphinx documentation? When I build the docs with `make html` I don't see where this link would show up.